timbermill-server/src/test/java/com/datorama/oss/timbermill/TimberLogLocalPersistenceTest.java

```diff
@@ -41,17 +38,6 @@ public static void init() throws IOException {
         TimberLogger.bootstrap(pipe, TEST);
     }
 
-    @After
-    public void check(){
-        ElasticsearchClient esClient = pipe.getEsClient();
```

Comment:
We need this check to make sure we actually went to the disk and not just worked regularly and wrote to ES without failing first
That's something that happened to us before, I fix that and added to this method